setup.py

```diff
@@ -53,24 +45,26 @@ def is_platform_mac():
 }
```

Comment:
If there are no objections, I'd like to move this dictionary up above the `min_cython_version = '0.28.2` line

Reply:
sure